homeassistant/components/light/zwave.py

```diff
@@ -244,9 +235,9 @@ def _value_added(self, value):
         # Check for the missing color values
         self._get_color_values()
 
-    def update_properties(self):
+    def update(self):
```

Comment:
I think balloob's conceern wasn't that a component would overwrite the zwave code, but that the zwave code would overwrite required code from the component. I agree with you though, that we don't have any current components that override update, and I can't imagine why they'd need to.

I agree that using `update` should be safe, but I also don't have strong feelings about what the method is named, so I'll defer to the consensus.